Sources/Common/Core/Math/index.js

```diff
@@ -2135,32 +2138,12 @@ export function float2CssRGBA(rgbArray) {
 }
 
 export function rotateVector(vectorToBeRotated, axis, angle) {
```

Comment:
If the previous code was wrong, it is likely that we have similar bug in VTK/C++...

I'm not a big fan for hiding how dev should use vtkMatrixBuilder for doing it on their end.
Also, since it is a new one, the identity() step is not needed.